### ISSUE TITLE
kitbgsv processes enter zombie state

### ISSUE THREAD

**Issue by mig5** [CollaboraOnline/online#9972]
### Describe the Bug

I am running Collabora as a Docker container (via the current `:latest` image tag with SHA `717b9b6a98d706219dfd6d81a33a27f908a9d31ae979b01748b6dad07b13b1e4`) alongside a Nextcloud 28 (also containerized) instance.

After some undetermined time where Collabora is used as part of Nextcloud to edit a document (e.g an .odt document), we notice a bunch of processes go into 'zombie' state on the system.

```
root@nextcloud:/home/mig5# ps aux | grep -e '[Z']
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
uuidd      19966  0.0  0.0      0     0 ?        ZN   Sep03   0:00 [kitbgsv_006_006] <defunct>
uuidd      20003  0.0  0.0      0     0 ?        ZN   Sep03   0:00 [kitbgsv_006_007] <defunct>
uuidd      20325  0.0  0.0      0     0 ?        ZN   Sep03   0:00 [kitbgsv_006_00e] <defunct>
uuidd      20436  0.0  0.0      0     0 ?        ZN   Sep03   0:00 [kitbgsv_006_010] <defunct>
uuidd      20544  0.0  0.0      0     0 ?        ZN   Sep03   0:00 [kitbgsv_006_012] <defunct>
uuidd      20560  0.0  0.0      0     0 ?        ZN   Sep03   0:00 [kitbgsv_006_013] <defunct>
uuidd      20908  0.0  0.0      0     0 ?        ZN   Sep03   0:00 [kitbgsv_006_017] <defunct>
uuidd      21473  0.0  0.0      0     0 ?        ZN   Sep03   0:00 [kitbgsv_006_019] <defunct>
uuidd      21836  0.0  0.0      0     0 ?        ZN   Sep03   0:00 [kitbgsv_018_001] <defunct>
```

(uuidd is just the enumeration of the user id as seen from the host running the Docker container)

The tree of processes related to the collabora container (showing the above processes associated):

```
  ├─containerd-shim,1604 -namespace moby -id b32975f8185151c07b4719c4916dc8a531ea4fba837422ad61787dd758b8cc86 -address /run/containerd/containerd.sock
  │   ├─{containerd-shim},1610
  │   ├─{containerd-shim},1611
  │   ├─{containerd-shim},1612
  │   ├─{containerd-shim},1613
  │   ├─{containerd-shim},1622
  │   ├─{containerd-shim},1628
  │   ├─{containerd-shim},1631
  │   ├─{containerd-shim},1632
  │   ├─{containerd-shim},1635
  │   ├─coolwsd,1717,cgroup,ipc,mnt,net,pid,uts --version --use-env-vars --o:sys_template_path=/opt/cool/systemplate --o:child_root_path=/opt/cool/child-roots ...
  │   │   ├─{coolwsd},2087
  │   │   ├─{prisoner_poll},2088
  │   │   ├─forkit,2089 --systemplate=/opt/cool/systemplate --lotemplate=/opt/collaboraoffice --childroot=/opt/cool/child-roots/1-70d77bb0/ --clientport=9980...
  │   │   │   ├─kit_spare_014,21349 --systemplate=/opt/cool/systemplate --lotemplate=/opt/collaboraoffice --childroot=/opt/cool/child-roots/1-70d77bb0/ --clientport=9980...
  │   │   │   ├─kit_spare_015,21365 --systemplate=/opt/cool/systemplate --lotemplate=/opt/collaboraoffice --childroot=/opt/cool/child-roots/1-70d77bb0/ --clientport=9980...
  │   │   │   ├─kit_spare_01a,21754 --systemplate=/opt/cool/systemplate --lotemplate=/opt/collaboraoffice --childroot=/opt/cool/child-roots/1-70d77bb0/ --clientport=9980...
  │   │   │   └─kit_spare_01b,21778 --systemplate=/opt/cool/systemplate --lotemplate=/opt/collaboraoffice --childroot=/opt/cool/child-roots/1-70d77bb0/ --clientport=9980...
  │   │   ├─{remotefontconfi},2097
  │   │   ├─{accept_poll},2098
  │   │   ├─{websrv_poll},2099
  │   │   ├─{admin},2100
  │   │   ├─(kitbgsv_006_006,19966)
  │   │   ├─(kitbgsv_006_007,20003)
  │   │   ├─(kitbgsv_006_00e,20325)
  │   │   ├─(kitbgsv_006_010,20436)
  │   │   ├─(kitbgsv_006_012,20544)
  │   │   ├─(kitbgsv_006_013,20560)
  │   │   ├─(kitbgsv_006_017,20908)
  │   │   ├─(kitbgsv_006_019,21473)
  │   │   └─(kitbgsv_018_001,21836)
  │   ├─{containerd-shim},2107
  │   └─{containerd-shim},2151
```

Some more context from the docker container in case relevant:

```
            "Labels": {
                "author": "Collabora Productivity Ltd.",
                "com.docker.compose.config-hash": "bb6550de0d3293452941f42c7828cacbcd45da0a72a66d485bc33cc060bd8add",
                "com.docker.compose.container-number": "1",
                "com.docker.compose.depends_on": "nextcloud_app:service_started:false",
                "com.docker.compose.image": "sha256:717b9b6a98d706219dfd6d81a33a27f908a9d31ae979b01748b6dad07b13b1e4",
                "com.docker.compose.oneoff": "False",
                "com.docker.compose.project": "nextcloud",
                "com.docker.compose.project.config_files": "/opt/nextcloud/docker-compose.yml",
                "com.docker.compose.project.working_dir": "/opt/nextcloud",
                "com.docker.compose.replace": "cbf8041c16f3d17a8e4148aceb11538f43c71813f0871f91b18ba19c4feec563",
                "com.docker.compose.service": "nextcloud_collabora",
                "com.docker.compose.version": "2.29.1",
                "commit.history.core": "https://git.libreoffice.org/core/+log/cp-24.04.5-2",
                "commit.history.online": "https://github.com/CollaboraOnline/online/commits/cp-24.04.5-2",
                "description": "Collabora Online is a powerful collaborative Office suite that supports all major document, spreadsheet and presentation file formats, which you can integrate into your own infrastructure. Collabora Online provides data security and sovereignty, and is ideally suited to the demands of a modern distributed working environment. Delivering a familiar look and feel, Collabora Online represents a real alternative to other big-brands solutions, giving you control and flexibility.",
                "release.notes": "https://www.collaboraoffice.com/code-24-04-release-notes/",
                "version": "24.04.5.2"
            }
```

I am not actually sure if it is a 'problem' as such, but it tripped our monitoring.

Restarting the container clears the zombies.

I wanted to raise it in case it was unexpected, maybe there's a way the container should be reaping its processes properly.

### Steps to Reproduce

Use collabora as a docker container with Nextcloud to work on an .odt document, check for zombie processes on system afterward.

### Expected Behavior

Didn't expect zombie processes to occur.

### Actual Behavior

Zombie processes occur.

### Desktop
(Please complete the following information)
 - **Collabora version:** 24.04.5.2
 - **OS and version:** Host running the container is Debian 12, docker 27.2.0
 - **Browser and version:** N/A

**Comment by husamash** [comment]
Have been facing the same issue for a few months with similar setup. Have been updating the collabora container to the latest image, currently at 24.04.8.2 and nextcloud at 29.0.8 and nextcloud office app at 8.4.8.

**Comment by Ashod** [comment]
This should be resolved in 24.04.8.3. The fix was in PR #10170.

Please confirm if the issue is resolved, or reopen with logs (debug or trace level), if not.

Thank you.
